components/webui/package.json

```
@@ -2,16 +2,15 @@
   "name": "webui",
   "version": "0.1.0",
   "description": "",
+  "workspaces": [
+    "common",
+    "client",
+    "server"
+  ],
   "scripts": {
-    "client:lint:check": "cd client && npm run lint:check",
-    "client:lint:fix": "cd client && npm run lint:fix",
-    "client:start": "cd client && npm start",
-    "init": "npm clean-install && (cd client && npm clean-install) && (cd server && npm clean-install)",
-    "lint:check": "npm run client:lint:check && npm run server:lint:check",
-    "lint:fix": "npm run client:lint:fix && npm run server:lint:fix",
-    "server:lint:check": "cd server && npm run lint:check",
-    "server:lint:fix": "cd server && npm run lint:fix",
-    "server:start": "cd server && npm start",
+    "init": "npm clean-install --workspaces",
+    "lint:check": "npm --workspace common run build && npm run lint:check --workspaces",
+    "lint:fix": "npm --workspace common run build && npm run lint:fix --workspaces",
     "start": "concurrently \"npm run client:start\" \"npm run server:start\""
```

let's fix this broken script

i believe a more elegant way is possible once we unify the dev script names, for now this
```suggestion
    "start": "concurrently \"npm run start:*\"",
    "start:client": "npm --workspace client run start",
    "start:common": "npm --workspace common run build:watch",
    "start:server": "npm --workspace server run dev"
```

also we should update the `concurrently` version to match the server's version (see below). otherwise, `concurrently` won't be installed due to version conflict.
```
    "concurrently": "^9.1.2"
```
then
```
rm -rf node_modules package-lock.json
npm i --workspaces
```